### PR TITLE
Exploratory data analysis

### PR DESCRIPTION
**Pull request by iqdecay**

### CODE REVIEW

File: eda_housing.ipynb [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+     "execution_count": 86,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Exploratory data analysis on Boston housing data\n",
+    "df = load_dataset(\"housing_data.csv\")\n",
+    "df = df.dropna()\n",
+    "print(df.head())\n",
```
Comment (robincourant) [inline eda_housing.ipynb]: Use `display` instead of `print`.

File: eda_housing.ipynb [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.5"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```
Comment (robincourant) [inline eda_housing.ipynb]: Add a correlation matrix plot like [this](http://jamesmarquezportfolio.com/correlation_matrices_in_r.html#.WP6WeFB-BPd.facebook).

File: eda_housing.ipynb [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# PLot the correlation of variables\n",
```
Comment (robincourant) [inline eda_housing.ipynb]: Typo : "PLot".

File: eda_housing.ipynb [hunk 4]
```diff
@@ -0,0 +1,385 @@
...
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 55,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from data_preparation import load_dataset,prepare_data\n",
+    "import pandas as pd\n",
+    "import seaborn as sb\n",
```
Comment (robincourant) [inline eda_housing.ipynb]: Add to requirements.

File: eda_housing.ipynb [hunk 5]
```diff
@@ -0,0 +1,385 @@
...
+      ]
+     },
+     "execution_count": 86,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Exploratory data analysis on Boston housing data\n",
+    "df = load_dataset(\"housing_data.csv\")\n",
```
Comment (robincourant) [inline eda_housing.ipynb]: Add a path variable, and please load the dataset folder on the drive in order to have same files and path.

**Review by robincourant — CHANGES_REQUESTED** [review]
- Download the dataset folder on the drive and use the same path and file names !
- Plot the correlation matrix as shown in my comment
- Fix little issues
- Do the same notebook with the other dataset
- Put the two notebook in one directory

File: eda_housing.ipynb (hunk 2 above)
Comment (iqdecay) [reply eda_housing.ipynb]: Quite impossible except by crafting, graph by graph, the grid plot. The function in link in an R package.

**Comment by iqdecay** [comment]
@robincourant

**Pull request merged**